src/site/content/en/blog/stories-component/index.md

```diff
@@ -135,43 +205,110 @@ With CSS grid, we can define a single cell grid (aka a square), where the rows a
 
-On [caniuse](https://caniuse.com/#search=scroll-snap-stop) it shows Chromium only at the time of writing this article. This means on Firefox, Safari, etc that a user can skip a friend. It's fine, user's won't be blocked by this. They may just need to be careful, or we can write js to ensure that friend they skipped isn't marked as viewed.
+Carousels were one of the major UX use cases that prompted the creation of the
+CSS Scroll Snap Points spec. Unlike Stories, A Carousel doesn't always need to stop
+on each image after a user interacts with it. It might be fine or encouraged to
+quickly cycle through the Carousel. Stories, on the other hand, are best navigated one-by-one,
+and that's exactly what `scroll-snap-stop` provides.
 
-[Read more in the spec](https://www.w3.org/TR/css-scroll-snap-1/#scroll-snap-stop) if you're interested.
+At the time of writing this post, `scroll-snap-stop` is only supported on Chromium-based
+browsers. You should check out
+[Browser compatibility](https://developer.mozilla.org/docs/Web/CSS/scroll-snap-stop#Browser_compatibility)
+for updates, though. It's not a blocker, though. It just means that on unsupported browsers
+users can accidentally skip a friend. So users will just have to be more careful, or
+we'll need to write JavaScript to ensure that a skipped friend isn't marked as viewed.
+
+Read more in [the
+spec](https://www.w3.org/TR/css-scroll-snap-1/#scroll-snap-stop) if you're
+interested.
 
 ### `overscroll-behavior`
-Ever had a modal scroll behind you or found times when scrolling seems to jump out of an inner scrolling space and into an outer scrolling space? `overscroll-behavior` let's the developer trap that scroll and never let it leave. It's nice for all sorts of occasions, and this stories component uses it to prevent additional swipes and scrolling gestures from leaving the stories component. 
 
-Safari and Opera were the 2 browser's that didn't [support](https://caniuse.com/#search=overscroll-behavior) this, and that's totally ok. Those users will get an overscroll experience like they're used to and may never notice this enhancement. I'm personally a big fan and like including it as part of nearly every overscroll feature I implement. It's a harmless addition that only can lead to an improvement.
+<!-- TODO(kayce): Video would be helpful. -->
+
+Have you ever been scrolling through a modal when all of a sudden you
+start scrolling the content behind the modal?
+[`overscroll-behavior`](https://developer.mozilla.org/docs/Web/CSS/overscroll-behavior) 
+lets the developer trap that scroll and never let it
+leave. It's nice for all sorts of occasions. My Stories component uses it
+to prevent additional swipes and scrolling gestures from leaving the
+component. 
```

Comment:
I think I mangled this section and changed the meaning. We need to figure out how to phrase it so it's easier to grok.